src/Microsoft.OData.Client/DataServiceRequestOfT.cs

```diff
@@ -58,7 +58,7 @@ internal DataServiceRequest(Uri requestUri, QueryComponents queryComponents, Pro
 
         #endregion Constructors
 
-        /// <summary>Gets the type of the object used to create the <see cref="T:Microsoft.OData.Client.DataServiceRequest`1" /> instance.</summary>
+        /// <summary>Gets the type of the object used to create the <see cref="Microsoft.OData.Client.DataServiceRequest`1" /> instance.</summary>

```

Comment:
I didn't resolve ALL warnings. I will split this across multiple pull requests, as otherwise it will get too big to review. There will still be warnings left after this pull request is completed. Either way, your comment is definitely valid, I added the change.